packages/client/src/pages/contribute/TaggingInterface.tsx

```diff
@@ -4,24 +4,22 @@ import { TagForm } from '../../components/contribute/TagForm.component';
 import { useStudy } from '../../context/Study.context';
 import { useEffect, useState } from 'react';
 import { useCompleteTagMutation } from '../../graphql/tag/tag';
-import { NoTagNotification } from '../../components/contribute/NoTagNotification.component';
 import { Study } from '../../graphql/graphql';
 import { TagProvider, useTag } from '../../context/Tag.context';
 
 export const TaggingInterface: React.FC = () => {
   const { study } = useStudy();
-
   // TODO: View for when there is no study vs when there is no tag
   return (
-    <>
-      <TagProvider>
+    <TagProvider>
```

Comment:
Only need one tag provider, since its in `ContributeLanding` it doesn't need to be in `TaggingInterface`